ajax/libs/vega-embed/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "vega-embed",
+  "filename": "vega-embed.min.js",
+  "version": "2.2.0",
+  "description": "Publish Vega visualizations as embedded web components with interactive parameters.",
+  "homepage": "https://github.com/vega/vega-embed",
```

Comment:
We'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo. Let's remove it. Thanks.